code/modules/mob/living/simple_animal/abnormality/aleph/nobody_is.dm

```diff
@@ -576,7 +578,7 @@
 			grab_victim.deal_damage(strangle_damage, BLACK_DAMAGE)
 		else	//Apply ramping damage
 			playsound(get_turf(src), 'sound/effects/wounds/crackandbleed.ogg', 200, 0, 7)
-			grab_victim.deal_damage((strangle_damage * (3 - count)), BLACK_DAMAGE)
+			grab_victim.deal_damage((strangle_damage * (count - 3)), BLACK_DAMAGE)
```

Comment:
Will this cause the overall damage to be in the negative? because if so you may want to use the CLAMP proc to keep it above 0.